src/js/components/ServiceDetailTaskTab.js

```diff
@@ -0,0 +1,37 @@
+import mixin from 'reactjs-mixin';
+import React from 'react';
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import MesosStateStore from '../stores/MesosStateStore';
+import Service from '../structs/Service';
+import TaskView from './TaskView';
+
+class ServiceDetailTaskTab extends mixin(StoreMixin) {
+
+  constructor() {
+    super(...arguments);
+
+    this.store_listeners = [
+      {name: 'state', events: ['success']}
```

Comment:
this adds a property that the `StoreMixin` will then use. this essentially makes the component listen to the `'state'` store for the `'success'` event (in which case it will then re-render)